lib/Extension/LanguageServerCodeTransform/CodeAction/PropertyAccessGeneratorProvider.php

```diff
@@ -16,23 +15,33 @@
 use Phpactor\WorseReflection\Reflector;
 use function Amp\call;
 
-class GenerateAccessorsProvider implements CodeActionProvider
+class PropertyAccessGeneratorProvider implements CodeActionProvider
 {
-    public const KIND = 'quickfix.generate_accessors';
+    private string $kind;
+
+    private string $command;
+
+    private string $type;
 
     private Reflector $reflector;
 
-    public function __construct(Reflector $reflector)
-    {
+    public function __construct(
+        string $kind,
+        string $command,
+        string $type,
```

Comment:
generatorRole ? but `type` is also kinda ok.

Reply:
I changed it to generatorRole